.ci/docker/common/install_conda.sh

```diff
@@ -52,12 +52,17 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
 
   # Install PyTorch conda deps, as per https://github.com/pytorch/pytorch README
   CONDA_COMMON_DEPS="astunparse pyyaml mkl=2021.4.0 mkl-include=2021.4.0 setuptools"
-  if [ "$ANACONDA_PYTHON_VERSION" = "3.11" ]; then
-    conda_install numpy=1.23.5 ${CONDA_COMMON_DEPS}
+  if [ "$ANACONDA_PYTHON_VERSION" = "3.11" ] || [ "$ANACONDA_PYTHON_VERSION" = "3.12" ]; then
+    conda_install numpy=1.26.0 ${CONDA_COMMON_DEPS}
   else
     conda_install numpy=1.21.2 ${CONDA_COMMON_DEPS}
   fi
 
+  if [ "$ANACONDA_PYTHON_VERSION" = "3.12" ]; then
```

Comment:
This is weird that this is only needed for 3.12?

Reply:
This comes from original PR:  https://github.com/pytorch/pytorch/pull/113987